go.mod

```diff
@@ -77,7 +77,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsperfcountersreceiver v0.69.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.69.0
 	github.com/signalfx/golib/v3 v3.3.47
-	github.com/signalfx/signalfx-agent v1.0.1-0.20230104182534-9eee411fe305
+	github.com/signalfx/signalfx-agent v1.0.1-0.20230117211433-9858481024e4
```

Comment:
The agent and apm replace directives also need to be updated since the signalfx exporter shares the dep.

Reply:
https://github.com/signalfx/splunk-otel-collector/blob/main/go.mod#L497-L498